repo/packages/Q/quobyte/2/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Yes, the `.cli` property allows configuration of native subcommands for different platforms.